cypress/integration/item/share/categories.spec.js

```diff
@@ -0,0 +1,34 @@
+import {
+  buildShareButtonId,
+  SHARE_ITEM_CATEGORY_LEVEL,
+} from '../../../../src/config/selectors';
+import { buildItemPath } from '../../../../src/config/paths';
+import {
+  ITEM_WITH_CATEGORIES,
+  SAMPLE_CATEGORIES,
+} from '../../../fixtures/categories';
+import { DEFAULT_TAGS } from '../../../fixtures/itemTags';
+
+const openShareItemTab = (id) => {
+  cy.get(`#${buildShareButtonId(id)}`).click();
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const changeCategory = (value) => {
+  cy.get(`#${SHARE_ITEM_CATEGORY_LEVEL}`).click();
+  cy.get(`li[data-value="${value}"]`).click();
+};
+
+describe('Categories', () => {
+  it('Item Categories', () => {
+    const item = ITEM_WITH_CATEGORIES;
+    cy.setUpApi({ items: [item], tags: DEFAULT_TAGS });
+    cy.visit(buildItemPath(item.id));
+    openShareItemTab(item.id);
+
+    // css selector
+    const categorySelect = cy.get('span.MuiChip-label');
+
+    categorySelect.contains(SAMPLE_CATEGORIES[0].name);
+  });
```

Comment:
Maybe you are working on it, but could we have these tests as well:
- adding a new category calls the endpoint
- having only reading access/logging out show disabled categories 
- delete a category calls the endpoint
- if the item does not have categories, it shows correctly. That said, it would be nice to not show the category section if it is empty -> no empty disabled input